src/platform/packages/shared/kbn-mcp-dev-server/src/tools/symbol_analysis.ts

```diff
@@ -139,10 +139,25 @@ async function findUsagesHandler(input: z.infer<typeof findUsagesInputSchema>) {
   };
 }
 
-export const findUsagesTool: ToolDefinition<typeof findUsagesInputSchema> = {
-  name: 'find_usages',
-  description:
-    "Analyzes a code symbol's usage across the entire codebase and generates a rich, categorized report. Use this tool to quickly understand a symbol's architectural role, differentiate between its definition, execution sites, and type declarations, and discover where it is referenced in tests and documentation. This is a primary tool for high-level code intelligence and analysis.",
+const description = `The precision tool for the second step in a "chain of investigation." Use this *after* \`semantic_code_search\` has helped you identify a specific, concrete symbol (e.g., a class name, function name, or type alias).
+
+**Best for:**
+*   **Drilling Down:** Answering the question, "Now that I've found \`IndicatorType\`, where is it actually used and how is it connected to the rest of the system?"
+*   **Architectural Analysis:** The rich, categorized report helps you understand a symbol's role by showing you:
+    *   Its definition.
+    *   Where it is imported and used (call sites).
```

Comment:
Just for my own understanding, I didn't understand the `(call sites)` part here is for